meta-genivi-dev/recipes-sota/genivi-swm/genivi-swm_git.bb

```diff
@@ -0,0 +1,60 @@
+DESCRIPTION = "Genivi Software Loading Manager"
+SECTION = "base"
+LICENSE = "MPL-2.0"
+LIC_FILES_CHKSUM = "file://${S}/LICENSE;md5=8d2127b230519ce4ad2c16070bdeb7b3"
+
+SRCREV = "cc05b2461c5fca8992f5b1f12ee81ac19698bf17"
+SRC_URI = "\
+    git://github.com/advancedtelematic/genivi_swm;branch=gdp-build;protocol=https \
+    file://package_manager.service \
+    file://partition_manager.service \
+    file://lifecycle_manager.service \
+    file://software_loading_manager.service \
+    file://module_loader_ecu1.service \
+    file://org.genivi.SoftwareLoadingManager.conf \
+    "
+
+PR="r0"
+
+S = "${WORKDIR}/git"
+
+FILES_${PN} = " \
+    /usr/lib/genivi-swm/* \
+    /lib/systemd/system/package_manager.service \
+    /lib/systemd/system/partition_manager.service \
+    /lib/systemd/system/module_loader_ecu1.service \
+    /lib/systemd/system/software_loading_manager.service \
+    /lib/systemd/system/lifecycle_manager.service \
+    /etc/dbus-1/system.d/org.genivi.SoftwareLoadingManager.conf \
+    "
+
+DEPENDS = "systemd"
+
+RDEPENDS_${PN} += " \
+    python-subprocess \
+    python-pygobject \
+    python-dbus \
+    python-storm \
+    squashfs-tools \
+    rpm \
+    "
+
+inherit systemd
+SYSTEMD_SERVICE_${PN} = " package_manager.service partition_manager.service module_loader_ecu1.service software_loading_manager.service lifecycle_manager.service "
+
+do_install () {
+    install -d ${D}${libdir}/${PN}
+
+    # TODO: Probably want to just copy in the needed files
+    cp -r ${S}/* ${D}${libdir}/${PN}
+
+    install -d ${D}${systemd_unitdir}/system
+    install -c "../package_manager.service" ${D}${systemd_unitdir}/system
```

Comment:
../package_manager.service ${D}${systemd_unitdir}/system  -> ${WORKDIR}/package_manager.service ${D}${systemd_system_unitdir}

And please change below lines